docs/graph_engine/from_networkx.rst

```diff
@@ -82,7 +82,7 @@ Create a client to use from the temp folder:
 
 .. code-block:: python
 
-	>>> cl = client.MemoryGraph(working_dir.name, [0])
+	>>> cl = client.MemoryGraph(working_dir.name)
```

Comment:
I think we should leave in the partitions [0] for this example so new users are aware of this parameter.

cl = client.MemoryGraph(working_dir.name, [0])

Reply:
Fewer parameters is better for cognitive load.

Reply:
Yes but if the user increases the converter num_workers > 1, they will not know how to load it.

Reply:
They can read the docstring to class explaining parameters and for advanced usage we still have examples in `spark_converter` an `custom_decoder` docs. This is a first doc with examples in our index and we add more complex things later.